schema/deploy/computed_columns/application_revision_naics_code.sql

```diff
@@ -13,7 +13,7 @@ create or replace function ggircs_portal.application_revision_naics_code(app ggi
         from ggircs_portal.form_result
         join ggircs_portal.form_json
         on form_result.form_id = form_json.id
-        and form_json.slug in ('admin', 'admin-2018')
+        and form_json.slug in ('admin', 'admin-2018', 'admin-2020')
```

Comment:
it looks like the `admin` slug is becoming the 2019 form for ever
I wonder if we should rename it as well, and maybe make "admin" behave like an alias through a computed column or something, to be the current year (or latest?) form in the system
Anyways, thoughts for tech debt :)

Reply:
actually, the slug could be a computed column built from form type + year

Reply:
I thought about it, but starting with this PR the slug is what is used in the `on conflict` clause when inserting the form schemas in the database, so we can't change it, unless we would change it live in prod just before deploying the release.

We could add some metadata to this table though, or update the form_json.description so that it actually says something useful :)

Reply:
true - we do have a 'description' column in that table that [we don't use for anything useful](https://github.com/bcgov/cas-ciip-portal/blob/f4ebd3c7df002b302f925118ad186a00eda5f6af/schema/data/prod/form_json.sql#L32)...yet!